dwave/embedding/pegasus.py

```diff
@@ -21,6 +21,47 @@
 import networkx as nx
 
 
+def _pegasus_fragment_helper(m=None, target_graph=None):
```

Comment:
I understand that this is a private function, but perhaps a comment on the two functions that this helper returns? I can imagine a dev stepping through the code and just wanting a quick overview of what is being returned.